esteid/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '2.3.0'
+__version__ = "2.4.0"
```

Comment:
I think we need to go to 3.0 since this contains some breaking changes.

Reply:
I suggest making a `2.99.0bXX` release, and then a 3.x release with the new approach, provided it gets approved :) 

After all these changes, I don't think this library is easily usable, as the "actions" are not covered with any tests/not documented, plus other things mentioned in the PR's description.

Reply:
I don't see a difference between 2.99 and 3 beta release. I understood that your plan was to go with the new approach for our projects anyway so making the 2.99 does not make sense.

Reply:
3-beta would mean that the functionality for 3 is there but isn't quite ready. In this case the new api isn't there even in beta.
Of course this isn't a big issue. But I'd rather make these versions a little more distinct.

Reply:
But yeah if we're fine with making a release of the #34 (which contains all stuff from this one) as 3-x then I guess this is out of the question.